src/main/resources/templates/fragments/bootstrap_nav_header.html

```diff
@@ -72,14 +72,14 @@
 
     <ul class="nav navbar-nav navbar-right">
       <li th:if="${isLoggedIn}">
-        <a th:text="${login}" th:href="'https://github.com/' + ${login}"></a>
+        <a th:text="${login}" th:href="'https://google.com/' + ${login}"></a>
       </li>
       <li th:if="${isLoggedIn}" th:text="'(' + ${role} + ')'">(Role)</li>
       <li th:if="${isLoggedIn}" class="dropdown user-nav">
         <a href="#" class="dropdown-toggle float-left" data-toggle="dropdown" role="button" aria-haspopup="true"
           aria-expanded="false">
-          <img th:src="'https://avatars2.githubusercontent.com/u/' + ${id} + '?v=3&amp;s=40'" height="20" width="20"
-            th:alt="${login}" />
+          <!-- <img th:src="'https://avatars2.githubusercontent.com/u/' + ${id} + '?v=3&amp;s=40'" height="20" width="20"
```

Comment:
You could create a model attribute in `AuthControllerAdvice.java` to get this from the token attributes (as attribute `picture`).